### ISSUE TITLE
Conda package

### ISSUE THREAD

**Issue by trevorcampbell** [tidymodels/tidyclust#167]
Thanks so much for producing the `tidyclust` library!

It would be fantastic to have a conda package for this library on Conda Forge (so that one can easily include `tidyclust` in a reproducible computing environment via a conda `environment.yml` file for example). Right now it seems most R packages on conda have `r-....` naming convention, so I guess `r-tidyclust` would fit alongside `r-tidymodels`, etc.

**Comment by EmilHvitfeldt** [comment]
Hello @trevorcampbell ! Thanks for the feature request, I don't know how this is normally done, so I'm gonna ask around and find out

**Comment by lescai** [comment]
is there any update on this? 😇

**Comment by EmilHvitfeldt** [comment]
I'm going to close this issue. We normally don't add packages to conda-forge.

As far as i can tell I'm not required to be part of this solution, so if this is still something that people want they can do it here: https://github.com/conda-forge/staged-recipes

**Comment by lescai** [comment]
Fair point that you’re not required to help. Worth noting that no maintainer ever is, and the R community generally manages a more collaborative tone even when declining.
